libs/template/themes/src/fulfillment.theme.ts

```diff
@@ -1,13 +1,20 @@
 import { Theme } from '@spryker-oryx/experience';
 import { fulfillmentIcons, materialDesignIcons } from '@spryker-oryx/resources';
-import { defaultBreakpoints } from '@spryker-oryx/themes/breakpoints';
+import { Size } from '@spryker-oryx/utilities';
 
 export const fulfillmentTheme: Theme = {
   name: 'fulfillment',
-  breakpoints: defaultBreakpoints,
+  breakpoints: {
+    [Size.Sm]: {
+      max: 414,
+    },
+    [Size.Md]: {
+      min: 415,
+    },
```

Comment:
we should use:

```ts
[Size.Sm]: {
    max: 767,
  },
  [Size.Md]: {
    min: 768,
  },
```

And add a container to the larger breakpoint of 414px. This supposed to ensure that if your on a small screen, you will not see bars next to the 414px container size.